apps/portal/src/content/docs/components/drawer.mdx

```diff
@@ -65,20 +96,598 @@ import { Drawer } from '@harnessio/ui/components'
+
+      <Drawer.Content>
+        <Drawer.Header>
+          <Drawer.Title>Title</Drawer.Title>
+        </Drawer.Header>
+
+        <Drawer.Body />
+
+        <Drawer.Footer>
+          <ButtonGroup>
+            <Drawer.Close asChild>
+              <Button variant="outline">Cancel</Button>
+            </Drawer.Close>
+          </ButtonGroup>
+        </Drawer.Footer>
+      </Drawer.Content>
+    </Drawer.Root>
+
+`}
+/>
+
+## API Reference
+
+### `Root`
+
+Contains [`Drawer.Trigger`](#trigger) and [`Drawer.Content`](#content) components.
+
+<DocsPage.PropsTable
+  props={[
+    {
+      name: "defaultOpen",
+      description:
+        "The open state of the drawer when it is initially rendered. Use when you do not need to control its open state.",
+      required: false,
+      value: "boolean",
+      defaultValue: "undefined",
+    },
+    {
+      name: "open",
+      description:
+        "When set, operate in controlled mode in combination with onOpenChange",
+      required: false,
+      value: "boolean",
+      defaultValue: "undefined",
+    },
+    {
+      name: "onOpenChange",
+      description: "Callback notified when open state changes",
+      required: false,
+      value: "(open: boolean) => void",
+      defaultValue: "undefined",
+    },
+    {
+      name: "modal",
+      description:
+        "The modality of the dialog. When set to true, interaction with outside elements will be disabled and only dialog content will be visible to screen readers.",
+      required: false,
+      value: "boolean",
+      defaultValue: "true",
+    },
+    {
+      name: "container",
+      description: "Specify a container element to portal the drawer into.",
+      required: false,
+      value: "HTMLElement",
+      defaultValue: "document.body",
+    },
+    {
+      name: "direction",
+      description: "Direction the drawer will originate from",
+      required: false,
+      value: "'right' | 'left' | 'top' | 'bottom'",
+      defaultValue: "right",
+    },
+    {
+      name: "onAnimationEnd",
+      description:
+        "Gets triggered after the open or close animation ends, it receives an open argument with the open state of the drawer by the time the function was triggered.",
+      required: false,
+      value: "(open: boolean) => void",
+      defaultValue: "undefined",
+    },
+    {
+      name: "dismissible",
+      description:
+        "When false dragging, clicking outside, pressing esc, etc. will not close the drawer. Use this in combination with the open prop, otherwise you won't be able to open/close the drawer.",
+      required: false,
+      value: "boolean",
+      defaultValue: "true",
+    },
+    {
+      name: "handleOnly",
+      description: "When true dragging will only be possible by the handle.",
+      required: false,
+      value: "boolean",
+      defaultValue: "false",
+    },
+    {
+      name: "nested",
+      description: "Whether the drawer is nested in another drawer",
```

Comment:
_SUPER_ minor, but some descriptions have a `.` at the end and some don't. I imagine we'll want to start displaying the description by default at some point, so it's worth being consistent now.